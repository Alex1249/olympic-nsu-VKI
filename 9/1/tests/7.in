theinputcontains several test cases the first line of the input 
is an integer which indicates the number of tests each test description 
consists of ten lines of input the first nine lines of a test will 
describe an initial configuration in the format shown in figure the 
6 xt line will contain a list of rotations ending with the value for 
$ ch test case your program should print one line if your grandpa is 
correct print yes grandpa otherwise print no you are wrong
