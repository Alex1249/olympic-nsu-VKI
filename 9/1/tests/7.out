1 - 4
2 - 16
3 - 13
4 - 16
5 - 15
6 - 5
7 - 8
8 - 2
9 - 3
11 - 1
13 - 1
16 - 1
